lib/trip/event.rb

```diff
@@ -214,6 +216,25 @@ def line?
   end
   # @endgroup
 
+  ##
+  # @return [Hash]
+  #  Returns a Hash object that can be serialized to JSON.
+  def as_json
+    {
+      "event" => name.to_s, "path" => path,
+      "lineno" => lineno, "method_id" => method_id.to_s,
+      "method_type" => (Module === @tp[:self]) ? "singleton_method" : "instance_method",
+      "module_name" => (Module === @tp[:self]) ? @tp[:self].name : @tp[:self].class.name
```

Comment:
Add `module_name` method to `Trip::Event`.

Reply:
https://github.com/0x1eef/trip.rb/commit/0b1f4b531f0744656ca4c884ae595c6dc2c3cbee